core/templates/dev/head/pages/exploration-player-page/layout-directives/correctness-footer.directive.html

```diff
@@ -438,4 +438,4 @@
     <i class="material-icons tick-symbol">&#xE86C;</i>
     <span class="inner-content protractor-test-correct-feedback">CORRECT!</span>
   </div>
-</div>
+</div>
```

Comment:
Missing newline at EOF. (Make sure to proofread your PR on GitHub Files Changed immediately after submission.)